src/main/java/cms/gov/madie/measure/config/TerminologyServiceConfig.java

```diff
@@ -0,0 +1,26 @@
+package cms.gov.madie.measure.config;
+
+import lombok.Getter;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.client.RestTemplate;
+
+@Getter
+@Configuration
+public class TerminologyServiceConfig {
+
+  @Value("${madie.terminology.service.base-url}")
+  private String baseUrl;
+
+  @Value("${madie.terminology.service.fetch-value-sets}")
+  private String valueSetsUrl;
+
+  @Value("${madie.terminology.service.validate-codes}")
+  private String validateCodeUrl;
+
+  @Bean
+  public RestTemplate terminologyRestTemplate() {
+    return new RestTemplate();
```

Comment:
I'm seeing 3 or 4 rest templates being registered with spring context. Should I create a task to refactor this to have one common bean?